staubli_val3_driver/val3/ros_server/dataReceiver.pgx

```diff
@@ -29,7 +29,7 @@
   // 3: decode body
   // 4: push motion into buffer
   // 5: send ACK
-  l_nMsgRecvState=0
+  l_nMsgRecvState=nMsgState["RecHeader"]
```

Comment:
Interesting. Is this best practice for defining named constants?

Reply:
I don't think I would necessarily call it "best practice", but I can't think of another way to make it clear what those numbers mean without using macros (sadly not possible ☹️).

Reply:
Another possibility here would be to use a local variable, I.E. `l_nRecHeader` which could be initialized at the start of each program. I'm not sure which one would be more clear.

Reply:
Personally I would find a variable clearer. The fact it wouldn't be an actual constant would not immediately be apparent, but indexing into a sequence with a string is further from a constant than a variable.

If you'd use ALL_CAPS for the variable name (as is customary in some other languages), it would even resemble a constant.

Reply:
Agreed, I'll plan to change these to local variables. It would be possible to use all caps, but that would be against the "standard" for naming variables in VAL 3.